src/classes/User.js

```diff
@@ -22,6 +22,12 @@ class User {
   decideToCook(recipeId) {
     if(!this.recipesToCook.includes(recipeId)){
       this.recipesToCook.push(recipeId)
+    } else {
+      this.recipesToCook.forEach((recipe, i) => {
```

Comment:
maybe we can DRY this up/find a way to use same functionality for toggleFavorites and decideToCook if we have time to refactor?